jobs/scripts/parallel-tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+sed -i 's/    mr.developer/    mr.developer\ngit-clone-depth = 10/' core.cfg
```

Comment:
Why 10 instead of 1? And why patch the config on the fly in a script?

Reply:
1 would be fine if all sources had a branch, but if they have a tag or a revision that's going to fail.

And actually it did, on jobs that use ``py3.cfg`` they need a bit more history as there are some sources that point to revisions rather than branches.